quarkus/server/build.gradle.kts

```diff
@@ -61,6 +61,12 @@ tasks.named("distZip") { dependsOn("quarkusBuild") }
 
 tasks.named("distTar") { dependsOn("quarkusBuild") }
 
+tasks.named("javadoc") { enabled = false }
```

Comment:
Better add a dummy java file in there. When we eventually publish Maven artifacts, the publication will fail without a javadoc and a sources jar.

Reply:
I see you added a package-info.java - that should be enough.
Maybe you have to tell javadoc to not fail on errors (different attribute)